cf/i18n/resources/ko-kr.all.json

```diff
@@ -4129,7 +4129,7 @@
   },
   {
     "id": "Terminate the running application Instance at the given index and instantiate a new instance of the application with the same index",
-    "translation": "주어진 색인에서 실행 중인 애플리케이션 인스턴스를 종료하고 애플리케이션의 새 인스턴스를 동일한 색인으로 인스턴스화합니다"
+    "translation": "주어진 색인에서 실행 중인 애플리케이션 인스턴스를 종료하고 애플리케이션의 새 인스턴스를 동일한 색인으로 인스턴스화합니다."
```

Comment:
This is the description of `restart-app-instance`. Please don't add a period at the end of a command description.